test_suite/run_pypit_tests.py

```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env python
-#
-'''
+"""
 This needs to be run in the TEST_SUITES directory.
 Usually found in a Dropbox.
-'''
+
+IMPORTANT: Output will be written to TST_PYPIT env variable
```

Comment:
Do we need to specify the env variable? For example, on line 31, (I think) we could use:

`redfile = os.path.realpath(__file__)+instr.lower()+'_'+setup.lower()+'.pypit'`

and similarly on line 48 (although, I don't know where the env variable `PYPIT_CALIBS` and `TST_PYPIT` point to.